src/commands/force/org/open.ts

```diff
@@ -51,7 +50,21 @@ export class OrgOpenCommand extends SfdxCommand {
       return output;
     }
     // we actually need to open the org
-    await this.checkLightningDomain(url);
+    try {
+      this.ux.startSpinner(messages.getMessage('domainWaiting'));
+      await checkLightningDomain(url);
+    } catch (err) {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access,  @typescript-eslint/no-unsafe-call
+      if (err.message && err.message.includes('timeout')) {
```

Comment:
```suggestion
      if (err.message?.includes('timeout')) {
```
evaluates to undefined (falsy to the `if`) when there is no message.